Gcp-bastion-iap-firewall-best-practices.md

```
@@ -0,0 +1,317 @@
+# Google Cloud セキュアネットワーク構築ガイド
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)